pkgs/build-support/libredirect/default.nix

```
@@ -1,6 +1,22 @@
-{ stdenv, lib, coreutils }:
+{ stdenv, runCommand, lib, coreutils }:
 
-stdenv.mkDerivation rec {
+if stdenv.hostPlatform.isStatic
+then throw ''
+  libredirect is not available on static builds.
+
+  Please fix your derivation to not depend on libredirect on static
+  builds, using something like following:
+
+    nativeBuildInputs =
+      lib.optional (!stdenv.hostPlatform.isStatic) libredirect;
```

```suggestion
      lib.optional (!stdenv.buildPlatform.isStatic) libredirect;
```
because you can only use libredirect on executables from the build platform in cross.